packages/expo-av/src/__stories__/components/AudioPlayer.tsx

```diff
@@ -0,0 +1,127 @@
+import { diff } from 'deep-object-diff';
+import { Asset } from 'expo-asset';
+import { Audio, AVPlaybackStatus } from 'expo-av';
+import React from 'react';
+import { StyleProp, ViewStyle } from 'react-native';
+
+import Player from './Player';
+
+type PlaybackSource =
+  | number
+  | {
+      uri: string;
+      overrideFileExtensionAndroid?: string;
+      headers?: {
+        [fieldName: string]: string;
+      };
+    }
+  | Asset;
+
+interface Props {
+  style?: StyleProp<ViewStyle>;
+  source: PlaybackSource;
+}
+
+interface State {
```

Comment:
Somebody would argue about using `interface` in this scenario. Technically these are `type`s and by default we can go with `type` instead of `interface` in almost every case. I'm open to criticism though 😁 cc @tsapeta